commune_app/models.py

```diff
@@ -1,3 +1,9 @@
-# from django.db import models
+from django.db import models
 
-# Create your models here.
+
+class Chore(models.Model):
```

Comment:
@EliasAndry  I think it will be better to create separate Django app for each module in this way you will be able to work on each module with fewer conflicts with the rest of the team

Reply:
kept it in same app..
but yeah i added tests and once the commune model is ready ill remove the comment aswell